internal/helm/repository/oci_chart_repository_test.go

```diff
@@ -156,13 +158,23 @@ func TestOCIChartRepoisitory_Get(t *testing.T) {
 			version:     ">2.0.0",
 			expectedErr: "could not locate a version matching provided version string >2.0.0",
 		},
+		{
+			name:     "shouldn't error out with trailing slash",
+			version:  "",
+			url:      "oci://localhost:5000/my_repo/",
+			expected: "1.0.0",
+		},
 	}
 
-	url := "oci://localhost:5000/my_repo"
+	testURL := "oci://localhost:5000/my_repo/"
 	for _, tc := range testCases {
+		if tc.url != "" {
+			testURL = tc.url
+		}
```

Comment:
I'm afraid that this creates a shared variable `testURL` across all the test cases. So, when we add a new test case at the end without any `url`, `testURL` will use the value from the previous case.
How about moving `testURL` to the top and add it in each of the test cases, and then use `tc.url` below wherever we need the URL?